scripts/target/mame/mess.lua

```diff
@@ -3828,11 +3828,20 @@ createMESSProjects(_target, _subtarget, "sony_news")
 files {
 	MAME_DIR .. "src/mame/drivers/news_68k.cpp",
 	MAME_DIR .. "src/mame/drivers/news_r3k.cpp",
+	MAME_DIR .. "src/mame/drivers/news_r4k.cpp",
 	MAME_DIR .. "src/mame/drivers/news_38xx.cpp",
+	MAME_DIR .. "src/devices/machine/spifi3.cpp",
+	MAME_DIR .. "src/devices/machine/spifi3.h",
```

Comment:
This doesn’t go here – you need to set the variable that triggers including it in liboptional.

Reply:
Somehow, I missed that.... :/